keepsorted/options.go

```diff
@@ -139,10 +139,17 @@ func key(f reflect.StructField) string {
 	return key
 }
 
-func parseBlockOptions(commentMarker, options string, defaults blockOptions) (blockOptions, error) {
+// optionWarnings are things that we found wrong with the option string but don't need to halt the entire keep-sorted operation.
+type optionWarnings []error
```

Comment:
Nit: Does this wrapper type actually make anything better? It mostly made me have to dig more to figure out what was going on. I feel like just using `[]error` (or `[]*Finding`) would have been simpler. 

IMO doing `errors.Join(warns...)` instead of `warns.err()` isn't really much different.